conductor-client/src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -1125,7 +1184,11 @@ public static String columnName( ResultSet rs ) throws SQLException {
         return Lists.newArrayList( getTextArray( rs, COLUMN_NAMES_FIELD ) );
     }
 
-    public static PostgresDatatype sqlDataType( ResultSet rs ) throws SQLException {
+    public static String sqlDataType( ResultSet rs ) throws SQLException {
```

Comment:
This will be just a string, right? Why not just `dataType`?

```suggestion
    public static String dataType( ResultSet rs ) throws SQLException {
```